package.json

```diff
@@ -1,20 +1,34 @@
 {
   "name": "hackapp-client",
-  "version": "0.0.1",
-  "description": "",
-  "main": "src/index.js",
-  "scripts": {
-    "start": "node src/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "3.3.1"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/evscott/hackapp-client.git"
+  "scripts": {
+    "start": "PORT=9090 react-scripts start",
```

Comment:
So, with this environment variable set, is the client now running on port 9090 correctly?

Reply:
Yes! Everything spins up correctly with the docker containers you set up! (Yay!)